Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM metabrainz/python:3.8
+FROM metabrainz/python:3.8-20210115
 
-ENV DOCKERIZE_VERSION v0.2.0
+ENV DOCKERIZE_VERSION v0.6.1
 RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
     && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
 
```

Comment:
while we're editing this, let's make a few improvements to the dockerfile:
* use pip install --no-cache-dir
* add requirements.txt and run pip install before copying . to /code

change the comment "Consul Template service is already set up with the base image." to be more descriptive and follow the comment that we have in LB a little more closely. Note that we shouldn't copy the LB process of adding a down file and using rc.local, as we only ever run one service here.

Add a 'down' file for cron, because it exists in the base image but we don't need to run any cron jobs.